kube.libsonnet

```diff
@@ -577,6 +577,16 @@
     assert std.length(rel_paths) == 0 : "paths must be absolute: " + rel_paths,
   },
 
+  ManagedCertificates(name): $._Object('networking.gke.io/v1beta1', 'ManagedCertificate', name, domains) {
+    spec: {
+      domains: domains,
```

Comment:
Note that in this library we're following the pattern to specify _only_ the `name` as argument, then working with field overrides to complete object's settings.

In addition to above, resulting construct would be:
```
  gke:: { 
    ManagedCertificate(name): $._Object("networking.gke.io/v1beta1", "ManagedCertificate", name) { 
      spec: { 
        domains: error "spec.domains array is required",
      },
      assert std.length(self.spec.domains) > 0 : "ManagedCertificate '%s' spec.domains array must not be empty" % self.metadata.name,
    },
  },
```